src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -65,8 +61,7 @@ void createStation() {
     @Test
     void createStationWithDuplicateName() {
         // given
-        Map<String, String> params = new HashMap<>();
-        params.put(PARAM_NAME_TEXT, "강남역");
+        Map<String, String> params = requestParamStationName("강남역");
         createStation(params);
```

Comment:
```suggestion
        createStation("강남역");
```

요렇게 합쳐도 좋을 것 같은데 어떻게 생각하시나용? :D